thoth/report_processing/pretty_report.py

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env python3
+# thoth-report-processing
+# Copyright(C) 2020 Francesco Murdaca, Sai Sankar Gochhayat
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Utils Create Markdown output from adviser report input."""
+
+import pandas as pd
+from typing import Dict, Any
+
+
```

Comment:
What about calling the file `adviser.py` where we have a `class Adviser` that contains all methods we can use on adviser report?

Reply:
That sounds reasonable, I will change it accordingly.